scripts/main.js

```diff
@@ -300,13 +286,12 @@ async function handleStateSelection() {
               .append($(`<p>${distShortcode}</p>`))
           )
           .append(
-            $('<div class="results-cell election-cell"></div>')
-              .append($(`<p>${electionCode}</p>`))
+            $('<div class="results-cell party-cell"></div>')
+              .append($(`<p>${party}</p>`))
           )
           .append(
             $('<div class="results-cell score-cell"></div>')
                 .append($(`<p>${ccScore}</p>`))
```

Comment:
```suggestion
                .append($(`<p>${ccScore}</p>`))
                .append($(`<p>${party}</p>`))
```
As per Karan's preference, we should replace the election numbers here with party affiliation.